File: takeMeFront/takeMeFront/src/types/VolunteerPointsRecord.ts
/**
 * 志愿者积分/薪酬记录
 */
export interface VolunteerPointsRecord {
  id: number
  volunteerId: number
  orderId: number
  points: number         // 本次获得积分
  type: 0 | 1            // 0=服务奖励 1=惩罚扣除
  description: string    // 记录说明（如“完成助洁服务+10积分”）
  createTime: string
}

File: takeMe/pojo/src/main/java/com/me/entity/VolunteerPointsRecord.java
package com.me.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@TableName("volunteer_points_record")
public class VolunteerPointsRecord {

    @TableId(type = IdType.AUTO)
    private Long id;

    private Long volunteerId;
    private Long orderId;
    private Integer points;
    private Integer type;
    private String description;

    private LocalDateTime createTime;
}